hybris/custom/klarnapaymentaddon/acceleratoraddon/web/src/com/klarnapayment/controllers/pages/checkout/KPPaymentController.java

```diff
@@ -98,6 +98,39 @@ public PaymentsSession updateSession(@RequestBody
 		return creditSessionData;
 	}
 
+	@RequestMapping(value = "/auth-callback", method = RequestMethod.POST)
+	public @ResponseBody String authCallback(@RequestBody
+	final KlarnaPaymentAuthCallbackResponse klarnaPaymentAuthCallbackResponse) throws ApiException, IOException
+	{
+		if (klarnaPaymentAuthCallbackResponse != null)
+		{
+			try
+			{
+				LogHelper.debugLog(LOG, "Going to save Authorization for Klarna Payment ");
+				LOG.info("klarnaPaymentAuthCallbackResponse Token POST "
+						+ (klarnaPaymentAuthCallbackResponse != null
+								? klarnaPaymentAuthCallbackResponse.getAuthorizationToken() + " "
+										+ klarnaPaymentAuthCallbackResponse.getSessionId()
+								: klarnaPaymentAuthCallbackResponse));
+				if (klarnaPaymentAuthCallbackResponse.getAuthorizationToken() != null)
+				{
+					kpPaymentCheckoutFacade.saveAuthorization(klarnaPaymentAuthCallbackResponse.getAuthorizationToken(), null, null);
+					//Create payment transaction
+					kpPaymentFacade.createPaymentTransaction();
+					LogHelper.debugLog(LOG, "Saved Authorization Token for Klarna Payment with Session Id :: "
```

Comment:
Debug logging in production code

Reply:
Debug logs have been removed
Commit can be found in https://github.com/klarna/SAP-Commerce-klarna-payments/commit/c3d029e9830e600544e03277455cf385023f4a5c